HomeWork_12/HW12.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime
+def timer():
+    start = datetime()
+    def inner():
+        return datetime() - start
+    return inner
+
+def funcname(func):
+    def inner(*args,**kwargs):
+        print(f"Функція {func.__name__}")
+        return func(*args,**kwargs)
+
+import json
+
+
+
+phonebook = {
+        "Aleks": "347638756",
+        "Fedor": "3784638754",
+        "Den": "2387463745"
+
+    }
+json_phonebook = json.dumps(phonebook)
+while True:
+    @timer
```

Comment:
Your code is broken because of these two lines. Decorators should decorate functions but not be called anywhere in code